docs: update "create" documentation (#621)
VeryGoodOpenSource/very_good_cli: site/docs/templates/federated_plugin.md

This template is for a plugin that follows the [federated plugin architecture][federated_plugin_docs].

## Usage

```sh
# Create a new Flutter plugin named my_flutter_plugin (all platforms enabled)
very_good create flutter_plugin my_flutter_plugin --desc "My new Flutter plugin"
```

## Providing supported platforms

> If the plugin is intended to support only certain platforms, pass the `platforms` option with a comma-separated list of the platforms to support.
> If `platforms` is omitted, all platforms are enabled by default.
> The values can be: `android`, `ios`, `web`, `macos`, `linux` and `windows`.
> 

Reviewer: ```suggestion
If you want your plugin to support only some platforms, pass the `platforms` option with a comma-separated list of the platforms you want to support.

If `platforms` is omitted, all platforms are enabled by default.

The values for platforms are: `android`, `ios`, `web`, `macos`, `linux`, and `windows`.

```
